src/components/UI/Filters.tsx

```diff
@@ -22,6 +22,7 @@ export default function Filters() {
   const range = useSelector((state: RootState) => state.firefly.rangeDetails.range);
   const accounts = useSelector((state: RootState) => state.accounts.accounts);
   const selectedAccountIds = useSelector((state: RootState) => state.accounts.selectedAccountIds);
+  const selectedBrandStyle = useSelector((state: RootState) => state.configuration.selectedBrandStyle);
```

Comment:
same comment here we need to default to `colors.brandStyleOrange`

Reply:
Updated it. 55b5dc9f0af99d1f85951d3704744a830f4324ee a22bf175aec8ba26e175b450add53bc58bd3e3dc